spec/recipes_spec/books/pl-u-physics/expected_output.xhtml

```diff
@@ -33044,20 +33044,9 @@ podstaw podane w tre&#x15B;ci zadania warto&#x15B;ci drogi i czasu:<span data-ty
</m:mi><m:mtext>&#x2192;</m:mtext></m:mover></m:mrow><m:mo>&#xD7;</m:mo><m:mrow><m:mover><m:mi>F</m:mi><m:mtext>&#x2192;</m:mtext></m:mover></m:mrow></m:mrow><m:mo>)</m:mo></m:mrow><m:mo>&#x22C5;</m:mo><m:mrow><m:mover><m:mi>D</m:mi><m:mtext>&#x2192;</m:mtext></m:mover></m:mrow></m:mrow><m:annotation-xml encoding="MathML-Content"><m:mrow><m:mo>(</m:mo><m:mrow><m:mrow><m:mover><m:mi>A</m:mi><m:mtext>&#x2192;</m:mtext></m:mover></m:mrow><m:mo>&#xD7;</m:mo><m:mrow><m:mover><m:mi>F</m:mi><m:mtext>&#x2192;</m:mtext></m:mover></m:mrow></m:mrow><m:mo>)</m:mo></m:mrow><m:mo>&#x22C5;</m:mo><m:mrow><m:mover><m:mi>D</m:mi><m:mtext>&#x2192;</m:mtext></m:mover></m:mrow></m:annotation-xml></m:semantics></m:math>,</li><li><m:math display="inline"><m:semantics><m:mrow><m:mrow><m:mo>(</m:mo><m:mrow><m:mrow><m:mover><m:mi>A</m:mi><m:mtext>&#x2192;</m:mtext></m:mover></m:mrow><m:mo>&#xD7;</m:mo><m:mrow><m:mover><m:mi>F</m:mi><m:mtext>&#x2192;</m:mtext></m:mover></m:mrow></m:mrow><m:mo>)</m:mo></m:mrow><m:mo>&#x22C5;</m:mo><m:mrow><m:mo>(</m:mo><m:mrow><m:mrow><m:mover><m:mi>D</m:mi><m:mtext>&#x2192;</m:mtext></m:mover></m:mrow><m:mo>&#xD7;</m:mo><m:mrow><m:mover><m:mi>B</m:mi><m:mtext>&#x2192;</m:mtext></m:mover></m:mrow></m:mrow><m:mo>)</m:mo></m:mrow></m:mrow><m:annotation-xml encoding="MathML-Content"><m:mrow><m:mo>(</m:mo><m:mrow><m:mrow><m:mover><m:mi>A</m:mi><m:mtext>&#x2192;</m:mtext></m:mover></m:mrow><m:mo>&#xD7;</m:mo><m:mrow><m:mover><m:mi>F</m:mi><m:mtext>&#x2192;</m:mtext></m:mover></m:mrow></m:mrow><m:mo>)</m:mo></m:mrow><m:mo>&#x22C5;</m:mo><m:mrow><m:mo>(</m:mo><m:mrow><m:mrow><m:mover><m:mi>D</m:mi><m:mtext>&#x2192;</m:mtext></m:mover></m:mrow><m:mo>&#xD7;</m:mo><m:mrow><m:mover><m:mi>B</m:mi><m:mtext>&#x2192;</m:mtext></m:mover></m:mrow></m:mrow><m:mo>)</m:mo></m:mrow></m:annotation-xml></m:semantics></m:math>,</li><li><m:math display="inline"><m:semantics><m:mrow><m:mrow><m:mo>(</m:mo><m:mrow><m:mrow><m:mover><m:mi>A</m:mi><m:mtext>&#x2192;</m:mtext></m:mover></m:mrow><m:mo>&#x22C5;</m:mo><m:mrow><m:mover><m:mi>F</m:mi><m:mtext>&#x2192;</m:mtext></m:mover></m:mrow></m:mrow><m:mo>)</m:mo></m:mrow><m:mo>&#x22C5;</m:mo><m:mrow><m:mo>(</m:mo><m:mrow><m:mrow><m:mover><m:mi>D</m:mi><m:mtext>&#x2192;</m:mtext></m:mover></m:mrow><m:mo>&#xD7;</m:mo><m:mrow><m:mover><m:mi>B</m:mi><m:mtext>&#x2192;</m:mtext></m:mover></m:mrow></m:mrow><m:mo>)</m:mo></m:mrow></m:mrow><m:annotation-xml encoding="MathML-Content"><m:mrow><m:mo>(</m:mo><m:mrow><m:mrow><m:mover><m:mi>A</m:mi><m:mtext>&#x2192;</m:mtext></m:mover></m:mrow><m:mo>&#x22C5;</m:mo><m:mrow><m:mover><m:mi>F</m:mi><m:mtext>&#x2192;</m:mtext></m:mover></m:mrow></m:mrow><m:mo>)</m:mo></m:mrow><m:mo>&#x22C5;</m:mo><m:mrow><m:mo>(</m:mo><m:mrow><m:mrow><m:mover><m:mi>D</m:mi><m:mtext>&#x2192;</m:mtext></m:mover></m:mrow><m:mo>&#xD7;</m:mo><m:mrow><m:mover><m:mi>B</m:mi><m:mtext>&#x2192;</m:mtext></m:mover></m:mrow></m:mrow><m:mo>)</m:mo></m:mrow></m:annotation-xml></m:semantics></m:math>.</li></ol></div></div>
```

Comment:
I checked the exercises in pdf and it looks good: 
![image](https://user-images.githubusercontent.com/55203598/186692705-7a7213ac-8d4d-4d65-ba18-f6615df24d8d.png)